src/query/list_file.c

```diff
@@ -4924,6 +4975,7 @@ qfile_initialize_list_cache (THREAD_ENTRY * thread_p)
       /* create */
       qfile_List_cache.n_hts = prm_get_integer_value (PRM_ID_XASL_CACHE_MAX_ENTRIES) + 10;
       qfile_List_cache.list_hts = (MHT_TABLE **) calloc (qfile_List_cache.n_hts, sizeof (MHT_TABLE *));
+      qfile_List_cache.free_list = (int *) calloc (qfile_List_cache.n_hts, sizeof (int));
```

Comment:
When do you free() the free_list allocated here? 
I couldn't find that code.

Reply:
I missed the codes to free at the finalize routine. Thanks.

Reply:
add free() in this function where error:

Reply:
Ok, I fixed it.